scripts/cnode-helper-scripts/cncli.sh

```diff
@@ -799,8 +804,271 @@ cncliPTsendslots() {
 
 #################################
 
+cncliEpochData() {
+
+  proc_msg="~ CNCLI epochdata table load started ~"
+
+  getNodeMetrics
+
+  if ! cncliDBinSync; then
+    echo ${proc_msg}
+    echo "CNCLI DB out of sync :( [$(printf "%2.4f %%" ${cncli_sync_prog})] ... check cncli sync service!"
+    exit 1
+  else
+    echo ${proc_msg}
+    echo
+    checkEpochSlot "${choose_func}"
+  fi
+
+}
+
+loadEpochData() {
+  [[ -z "${POOL_VRF_SKEY}" ]] && POOL_VRF_SKEY="${POOL_DIR}/${POOL_VRF_SK_FILENAME}"
+  [[ -z "${CNCLI_DIR}" ]] && CNCLI_DIR="${CNODE_HOME}/guild-db/cncli"
+  [[ -z "${BLOCKLOG_DIR}" ]] && BLOCKLOG_DB="${CNODE_HOME}/guild-db/blocklog"
+
+  CNCLI_DB="${CNCLI_DIR}/cncli.db"
+  BLOCKLOG_DB="${BLOCKLOG_DIR}/blocklog.db"
```

Comment:
These are already defined by cncliInit, better to call that instead like all the other commands